cla_backend/apps/core/drf/router.py

```diff
@@ -1,21 +1,15 @@
-from collections import namedtuple, OrderedDict
-
+from collections import OrderedDict
 from django.core.exceptions import ImproperlyConfigured
 from django.core.urlresolvers import NoReverseMatch
-
 from rest_framework import views
 from rest_framework.routers import BaseRouter, flatten, replace_methodname
 from rest_framework.urlpatterns import format_suffix_patterns
-from rest_framework.compat import url
+from django.conf.urls import url
 from rest_framework.reverse import reverse
 from rest_framework.response import Response
 
 from rest_framework_nested.routers import NestedSimpleRouter as OriginalNestedSimpleRouter
```

Comment:
did you look to see if the parts in this file that are `Borrowed directly from DRF v3.0.2.` could be removed?

Reply:
Understood